crates/bindings-csharp/BSATN.Runtime/BSATN/Runtime.cs

```diff
@@ -175,7 +175,7 @@ public void Write(BinaryWriter writer, Inner? value)
         new AlgebraicType.U64(default);
 }
 
-public readonly struct U128 : IReadWrite<SpacetimeDB.U128>
+public readonly struct U128Stdb : IReadWrite<SpacetimeDB.U128>
```

Comment:
Why not call this one U128 and the other UInt128?
Or make both explicit by naming the other U128Sys?